js/ad_manager_vast.js

```diff
@@ -407,18 +432,22 @@ OO.Ads.manager(function(_, $) {
      * @returns {object} null if a VAST tag is absent, or if there are multiple VAST tags. Otherwise,
      * returns the VAST root element.
      */
-    var getVastRoot = _.bind(function(vastXML) {
-      var vastRootElement = $(vastXML).find("VAST");
-      if (vastRootElement.length === 0) {
-        _tryRaiseAdError("VAST: No VAST tags in XML");
-        return null;
-      }
-      else if (vastRootElement.length > 1) {
-        _tryRaiseAdError("VAST: Multiple VAST tags in XML");
+    var getVastRoot = function(vastXML) {
+      try {
+        var vastRootElement = vastXML.querySelectorAll("VAST");
+        if (vastRootElement.length === 0) {
+          _tryRaiseAdError("VAST: No VAST tags in XML");
+          return null;
+        }
+        if (vastRootElement.length > 1) {
+          _tryRaiseAdError("VAST: Multiple VAST tags in XML");
+          return null;
+        }
+        return vastRootElement[0];
+      } catch (error) {
```

Comment:
Throw a warning with the content of an error here, please. Suppressed exceptions are hard to debug.